cookbooks/aws-parallelcluster-environment/files/custom_action_executor/custom_action_executor.py

```diff
@@ -191,9 +192,21 @@ async def download_and_execute_scripts(self, scripts):
             *[self._download_script(script, idx) for idx, script in enumerate(scripts, 1)]
         )
         for script in downloaded_scripts:
+            logging.info("The hash of script %s is: %s", script.url, self._hash_of_file(script.path))
             await self._execute_script(script)
             os.unlink(script.path)
 
+    @staticmethod
+    def _hash_of_file(path):
```

Comment:
When adding new methods is useful to add unit tests to be sure it works as expected and avoid reducing code coverage.